docs/data-sources/instances.md

```diff
@@ -103,7 +103,7 @@ data "civo_instances" "small-with-backups" {
 * `cpu_cores` - Total cpu of the inatance.
 * `ram_mb` - Total ram of the instance.
 * `disk_gb` - The size of the disk.
-* `public_ip_requiered` - This should be either false, true or `move_ip_from:intances_id`.
+* `public_ip_requiered` - This should be either `create`, `none` or `move_ip_from:intances_id`.
```

Comment:
@alejandrojnm isn't the field actually `public_ip` rather than `public_ip_required`? (or even the misspelt here `public_ip_requiered` 😉 ) Definitely should be `create` or anything not `create` (so `none` works fine).

Reply:
According to my terraform plan call

![image](https://user-images.githubusercontent.com/38325136/121858741-3732da00-ccf7-11eb-89a1-65fb45f291ed.png)

i would thing that public_ip is the actuall ip.

Reply:
But then i trust you @andyjeffries more, as the owners of the provider 😓

Reply:
This may be a Go-thing, so @alejandrojnm will know more. I'm looking at the raw API itself, but maybe there's some mapping happening for the Go library and therefore Terraform.